services/docker/docker-hub-common-fetch.spec.js

```diff
@@ -0,0 +1,20 @@
+import sinon from 'sinon'
+import { expect } from 'chai'
+import { fetch } from './docker-hub-common-fetch.js'
+
+describe('fetch', function () {
+  it('invokes withJwtAuth', async function () {
+    const serviceInstance = {
+      _requestJson: sinon.stub().resolves('fake-response'),
+      authHelper: {
+        withJwtAuth: sinon.stub(),
+      },
+    }
+
+    const resp = await fetch(serviceInstance, {})
+
+    expect(serviceInstance.authHelper.withJwtAuth.calledOnce).to.be.true
```

Comment:
All I am doing is asserting that the fetch method calls the service class's `withJwtAuth` method, but I think that is all we care about. I don't think more elaborate mocks actually buy us much here.